packages/suite/src/support/discoveryHook.ts

```
@@ -0,0 +1,200 @@
+            reject: (error: Error) => void;
+        };
+    }[] = [];
+
+    const registerHook = (type: 'complete' | 'auth', deviceToSelect: TrezorDevice) => {
+        let selectorCallbacks:
+            | {
+                  resolve: (params: {
+                      state: Discovery['deviceState'];
+                      device: TrezorDevice;
+                  }) => void;
+                  reject: (error: Error) => void | undefined;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)